src/azure-cli/azure/cli/command_modules/acs/_params.py

```diff
@@ -217,6 +217,11 @@ def load_arguments(self, _):
         c.argument('enable_ahub', options_list=['--enable-ahub'])
         c.argument('node_osdisk_diskencryptionset_id', type=str, options_list=['--node-osdisk-diskencryptionset-id', '-d'])
         c.argument('aci_subnet_name')
+        c.argument('appgw_name', options_list=['--appgw-name'])
+        c.argument('appgw_subnet_cidr', options_list=['--appgw-subnet-cidr'])
+        c.argument('appgw_id', options_list=['--appgw-id'])
+        c.argument('appgw_subnet_id', options_list=['--appgw-subnet-id'])
+        c.argument('appgw_watch_namespace', options_list=['--appgw-watch-namespace'])
```

Comment:
Suggest to add`arg_group='Application Gateway'` for these options so that they will be shown under the `Application Gateway` titled group in `--help`. There are too many options in this command now.